player.c

```diff
@@ -1018,6 +1020,8 @@ void player_init(const struct player_callbacks *callbacks)
 #endif
 	pthread_attr_t *attrp = NULL;
 
+	cmus_mutex_init_recursive(&player_info.mutex);
```

Comment:
@mahkoh please do not forget about this one, as the code is non-obvious without the comment.